internal/error.go

```diff
@@ -137,6 +137,8 @@ type (
 		//
 		// NOTE: This option is supported by Temporal Server >= v1.24.2 older version will ignore this value.
 		NextRetryDelay time.Duration
+		// Category of the error. Maps to logging/metrics behaviours.
```

Comment:
```suggestion
		// Category of the error. Maps to SDK logging/metrics behaviours.
```
This `Category` does not effect server side metrics right?

Reply:
Correct

Reply:
Can you update the comment then if this is correct?

Reply:
Done